src/ad-sizes.ts

```diff
@@ -88,6 +88,9 @@ type SlotName =
 
 type Breakpoint = 'mobile' | 'desktop' | 'phablet' | 'tablet';
 
+const isBreakpoint = (s: string): s is Breakpoint =>
```

Comment:
This feels like it should be in `/lib` rather than being exported from here?

Reply:
Good idea. I think the type and this function should stay close together so I've moved both to a file in lib.